jobclient/python/tests/integration/test_client.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) Two Sigma Open Source, LLC
+#
+# Licensed under the Apache license, Version 2.0 (the "License");
+# you may not use this file ecept in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from unittest import TestCase
+
+from cook.scheduler.client import JobClient
+from cook.scheduler.client.jobs import Status as JobStatus
+
+CLIENT_HOST = os.environ.get('TEST_CLIENT_HOST', 'localhost:12321')
```

Comment:
The pattern we use for retrieving the URL under test is:

```
    @classmethod
    def setUpClass(cls):
        cls.cook_url = util.retrieve_cook_url()
        util.init_cook_session(cls.cook_url)

    def setUp(self):
        self.cook_url = type(self).cook_url
        ...
```

(e.g. https://github.com/twosigma/Cook/blob/master/integration/tests/cook/test_basic.py#L28-L34)